style.css

```diff
@@ -100,3 +113,214 @@ header {
   height: 1rem;
   margin: 0 0 0 16px;
 }
+
+.contact {
+  width: 327px;
+  height: 102px;
+  margin: auto;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact-title {
+  margin: 70px auto 16px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding: 1rem 0 0;
+  text-align: center;
+  color: #fff;
+}
+
+.container {
+  padding-top: 5rem;
+}
+
+.text-field {
+  height: 48px;
+  width: 327px;
+  margin: 0;
+  text-align: center;
+}
+
+#fname {
+  height: 48px;
+  width: 340px;
+  border-radius: 8px;
+}
+
+#lname {
+  margin-top: 1rem;
+  height: 48px;
+  width: 340px;
+  border-radius: 8px;
+}
+
+.subject {
+  margin: 0;
+}
+
+#subject {
+  border-radius: 8px;
+  height: 100px;
+  width: 340px;
+  margin-top: 2rem;
+  color: #b3bac5;
+}
+
+.btn {
+  font-family: sans-serif;
+  width: auto;
+  padding: 0 0.6rem;
+  height: 48px;
+  font-weight: 0;
+  font-size: 19px;
+  line-height: 24px;
+  border-radius: 10px;
+  border: 1px solid #dfe1e6;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background: #fff;
+  margin: 12px 0;
+  color: #396df2;
+  border-color: #6070ff;
+}
+
+.btn:hover {
+  box-shadow: 0 8px 16px #493df0;
+  border-radius: 8px;
+  color: #fff;
+  background-color: #6070ff;
+}
+
+.btn:enabled {
+  background-color: #fff;
+  color: #6070ff;
+}
+
+.btn:disabled {
+  background-color: #e5e5e5;
+  color: #5e6c84;
+  cursor: not-allowed;
+}
+
+.btn-big {
+  border-color: #6070ff;
+}
+
+.submit-btn:hover {
+  background-color: #3244e7;
+}
+
+.btn:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.cards .card {
+  border-radius: 16px;
+  margin: 6rem auto 0;
+  height: auto;
+  width: 90%;
+  padding: 16px;
+  background: #fff;
+}
+
+.cardss {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: 596px 596px 596px 596px;
+  grid-gap: 350px;
+}
+
+.img {
+  flex: none;
+  padding: 0;
+  margin: auto;
+  height: 280px;
+  width: 100%;
+  border-radius: 8px;
+  margin-top: 5px;
+}
+
+.works {
+  display: flex;
+  align-items: center;
+}
+
+.works2 {
+  display: flex;
+  align-items: center;
+}
+
+li {
+  color: #7a869a;
+}
+
+.works2 li {
+  margin-right: 0.5rem !important;
+}
+
+.works li {
+  margin-right: 0.3rem !important;
+}
```

Comment:
- [ ] [OPTIONAL] The use of `!important` to force styles in document is not considered best practice and it's better to avoid using this in order to reduce the probability of conflicts in our code. It might be okay to remove it and use more distinct `class` or `id` to select your elements.

Reply:
Thanks for the review @AbayomiOlaoye